test/Interop/C/stdlib/include-signal.swift

```diff
@@ -0,0 +1,3 @@
+// RUN: %target-typecheck-verify-swift -I %S/Inputs
+
+import IncludeSignal
```

Comment:
Please add an explanation to the test, what it is testing, what's the corner case, why it is difficult.

Reply:
Done.